<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="B_Add">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="B_upd">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>80</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="B_del">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>150</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>10</y>
     <width>461</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="B_upd_t">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>500</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Обновить Таблицу</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
